perception/autoware_traffic_light_multi_camera_fusion/launch/traffic_light_multi_camera_fusion.launch.xml

```diff
@@ -3,10 +3,11 @@
   <arg name="input/vector_map" default="/map/vector_map"/>
   <arg name="param_path" default="$(find-pkg-share autoware_traffic_light_multi_camera_fusion)/config/traffic_light_multi_camera_fusion.param.yaml"/>
   <arg name="output/traffic_signals" default="/perception/traffic_light_recognition/traffic_signals"/>
+  <arg name="all_traffic_light_camera" default="[camera6, camera7]"/>
```

Comment:
Is there any reason to put these parameter out here?

Reply:
The reason I put it in xml is to be able to start the node by itself with the following command. (If you don't put it in xml, it will fail when you launch it by itself.)
```
ros2 launch autoware_traffic_light_multi_camera_fusion traffic_light_multi_camera_fusion.launch.xml
```

I added this because I thought it was not good to have it fall down when launched by itself.


When launching Autoware as a whole, the parameters of tier4_perception_component.launch.xml will be inherited and used as before, so it is assumed to be unaffected.
https://github.com/autowarefoundation/autoware_launch/blob/caed5fd7576f2c348517fa5[...]_launch/launch/components/tier4_ perception_component.launch.xml